File: qlty-cli/tests/lang/scala/basic.in/FunctionComplexity.scala
class FunctionComplexity {
  def complex(a: Int, b: Int, c: Int): Int = {
    if (a > 0) {
      if (b > 0) {
        if (c > 0) {
          if (a > b) {
            if (b > c) {
              if (a + b > 10) {
                if (a * b > 100) {
                  return a
                }
              }
            }
          }
        }
      }
    }
    if (a < 0) {
      if (b < 0) {
        return b
      }
    }
    0
  }
}

File: qlty-cli/tests/lang/elixir/basic.in/FunctionComplexity.ex
defmodule FunctionComplexity do
  def process(items, flag, other) do
    for item <- items do
      if flag do
        case item do
          {:ok, value} ->
            if value > 0 and other do
              if value > 10 or flag do
                :big
              else
                :small
              end
            else
              :zero
            end

          {:error, reason} ->
            case reason do
              :timeout ->
                if other do
                  :retry
                else
                  :fail
                end

              _ ->
                :fail
            end

          _ ->
            :skip
        end
      else
        :disabled
      end
    end
  end
end
